spec/features/ips/remove_an_ip_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'features/support/sign_up_helpers'
+
+describe 'Remove an IP' do
+  let!(:user) { create(:user, :confirmed) }
```

Comment:
I share this concern, I think this spec will be clearer without `let!`.

Reply:
The reason I've done it this way is because I need access to the `ip` variable in the expectations, and this ip needs to belong to the user's organisation. Doing this with a before block looks more confusing to me, eg:

```
before do
  user = create(:user)
  sign_in_user user
  create(:location, organisation: user.organisation)
  create(:ip, location: location)
end

  expect(page).to have_content(Ip.first.address)
```